src/Workspaces/CSharp/Portable/CodeGeneration/ConstructorGenerator.cs

```diff
@@ -30,7 +30,7 @@ private static MemberDeclarationSyntax LastConstructorOrField(SyntaxList<MemberD
             IList<bool> availableIndices)
         {
             var constructorDeclaration = GenerateConstructorDeclaration(
-                constructor, options, destination?.SyntaxTree.Options ?? options.ParseOptions);
+                constructor, options, options.ParseOptions);
```

Comment:
we probably want to flip the order here.

Reply:
: ) I just want to test if reversing this would cause any test error. This is not the final fix